.travis/osx_runner.sh

```diff
@@ -64,6 +66,25 @@ script() {
     sudo make run
     mkdir -p logs
     sleep 1
+
+    #test grpc proxy
+    curl http://127.0.0.1:9080/apisix/admin/routes/1 -X PUT -d '
+    {
+        "methods": ["POST", "GET"],
+        "uri": "/helloworld.Greeter/SayHello",
+        "service_protocol": "grpc",
+        "upstream": {
+            "type": "roundrobin",
+            "nodes": {
+                "127.0.0.1:50051": 1
+            }
+        }
+    }'
```

Comment:
ditto

Reply:
need to add more test cases for error.

Reply:
> need to add more test cases for error.

can not make error, that will make travis fail

Reply:
I think we can write these test cases in lua, go or python, as we only call API of APISIX.

Reply:
We can this later, I have created a related issue: https://github.com/apache/incubator-apisix/issues/777